src/chrome/content/rules/corrupt-net.org.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="corrupt-net.org (partial)">
```

Comment:
Please add a hint about the invalid cert on `corrupt-net.org` and `www.corrupt-net.org`